### ISSUE TITLE
PostCSS 8

### ISSUE THREAD

**Issue by cl1ck** [giuseppeg/styled-jsx-plugin-postcss#42]
Can you please move postcss to peerDependencies? The direct dependency prevents using PostCSS 8 (and corresponding plugins).

**Comment by cl1ck** [comment]
I opened pull request #43 as there were some additional issues.

**Comment by surjithctly** [comment]
@cl1ck I'm getting the following error with Next JS using Postcss 8. Do you think your PR will fix it? 

```
[styled-jsx] Loading plugin from path: \node_modules\styled-jsx-plugin-postcss\index.js
Next.js 9 default postcss support uses a non standard postcss config schema https://err.sh/next.js/postcss-shape, 
you must use the interoperable object-based format instead https://nextjs.org/docs/advanced-features/customizing-postcss-config
```

**Comment by cl1ck** [comment]
@surjithctly Yes it should. The problem is that the plugin is still using PostCSS 7 as it's a direct dependency.

**Comment by reezpatel** [comment]
It is breaking for all NextJS Build v9.2.1+ as it enforces new postcss shape.

**Comment by reezpatel** [comment]
@giuseppeg Can you publish this?

**Comment by DaniGuardiola** [comment]
I encountered this as well and would love to see this merged. Will probably use it from the git fork directly in the meantime. Please @giuseppeg merge and publish this :)

**Comment by giuseppeg** [comment]
Thank you @cl1ck for the contribution and sorry everybody for the delay – I don't have a lot of time to contribute to this project. 

I just released 4.0.0
